nimbus/src/nimbus.idl

```diff
@@ -66,7 +66,8 @@ enum Error {
     "JSONError", "EvaluationError", "InvalidExpression", "InvalidFraction",
     "TryFromSliceError", "EmptyRatiosError", "OutOfBoundsError","UrlParsingError",
     "RequestError", "ResponseError", "UuidError", "InvalidExperimentFormat",
-    "InvalidPath", "InternalError", "NoSuchExperiment", "NoSuchBranch", "BackoffError"
+    "InvalidPath", "InternalError", "NoSuchExperiment", "NoSuchBranch", "BackoffError",
+    "InitializeNotYetComplete",
```

Comment:
Noticing without recommendation: we have begun to stop using the `Error` suffix.